tests/test_demos.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ $3 == "true" ];
```

Comment:
Sorry to nitpick on the formatting, lol:
```suggestion
if [ "$3" == "true" ]; then
    export VERSION=nightly
fi
```

Reply:
no worries at all, it's actually better to format it that way 👍

Reply:
done